utils/browser/src/lib.rs

```diff
@@ -173,12 +173,8 @@ impl Client {
 		});
 		wasm_bindgen_futures::future_to_promise(async {
 			match rx.await {
-				Ok(fut) => {
-					fut.await
-						.map(|s| JsValue::from_str(&s))
-						.ok_or_else(|| JsValue::NULL)
-				},
-				Err(_) => Err(JsValue::NULL)
+				Ok(fut) => fut.await.map(|s| JsValue::from_str(&s)).ok_or_else(|| JsValue::NULL),
```

Comment:
I'd add some space _somewhere_ when doing this one liner myself, like `.map( |s| JsValue::from_str(&s) )..`, but that's probably not substrate's style.

Reply:
Yeah, it's not IMO and this isn't a setting we can set afaik

Reply:
It looks hard to do well, but i created an issue anyways.. https://github.com/rust-lang/rustfmt/issues/4859

Reply:
If the one liner is too long we can reduce max line length?